documentation/src/main/asciidoc/userguide/chapters/domain/basic_types.adoc

```diff
@@ -428,11 +439,8 @@ include::{sourcedir}/basic/BooleanMappingTests.java[tags=basic-boolean-example-e
 ----
 ====
 
-Here the built-in `AttributeConverter` implementations provided by Hibernate are used.  A custom
-`AttributeConverter` could be used also.
-
-
-Booleans can also be mapped using `UserType` and `@CustomType`
+Of course, if you need to do something really special, a boolean value can also be mapped using
```

Comment:
Personally I think my original wording here is better.  There is nothing special or not here - its a user choice

Reply:
The reason I changed it was because (if I'm completely naive) it reads like this is something special about `Boolean's. But it's not: _any_ type can be mapped using `UserType`. So then I as the naive reader is left wondering what it is that is special about booleans that makes it especially likely that I would want to use `UserType` for them. We don't really say.

So I guess the best thing is to explain exactly *why* booleans are special in this way: and I agree my change did a bad job of that.

We should explain properly that there are all kinds of different encodings that you might find in a legacy database, and that our built-in `AttributeConverters` can't hope to handle all of them. And so if you have something they can't handle, then use a `UserType`.